cms/utils/reversion_hacks.py

```diff
@@ -1,6 +1,26 @@
 # -*- coding: utf-8 -*-
-import reversion
-from reversion.revisions import RegistrationError, VersionAdapter
+
```

Comment:
It looks like you're importing the same stuff even on ImportError, it's best to explicitly target the imports that will fail and add some comment as to why they would fail ie. "Reversion >= 1.x compat".

Also the revision manager assignments can easily be outside of the ImportError try/except and placed on their own try/except that catches the AttributeError, more specifically only the revision_manager seems to change location